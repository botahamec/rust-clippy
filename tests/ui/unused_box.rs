#![warn(clippy::unused_box)]

fn main() {
    // test code goes here
}
