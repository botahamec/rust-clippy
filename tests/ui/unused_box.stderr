error: function returns `Box<usize>` when `usize` implements `Sized`
  --> $DIR/unused_box.rs:3:13
   |
LL | fn foo() -> Box<usize> {
   |             ^^^^^^^^^^ help: change the return type to: `usize`
   |
   = note: `-D clippy::unused-box` implied by `-D warnings`

error: aborting due to previous error

